CHANGELOG.md

```diff
@@ -58,6 +58,12 @@
   * `go_cpu_classes_gc_total_cpu_seconds_total`
   * `go_cpu_classes_total_cpu_seconds_total`
   * `go_cpu_classes_idle_cpu_seconds_total`
+* [ENHANCEMENT] All: Add support for cluster validation in gRCP calls. When it is enabled, gRPC server verifies if a request coming from a gRPC client comes from an expected cluster. This validation can be configured by the following experimental configuration options: #10767
+  * `-server.cluster-validation.label`
+  * `-server.cluster-validation.grpc.enabled`
+  * `-server.cluster-validation.grpc.soft-validation`
+* [ENHANCEMENT] All: Add `cortex_client_request_invalid_cluster_validation_labels_total` metrics, that is used by Mimir's gRPC clients to track invalid cluster validations. #10767
+* [ENHANCEMENT] Ingester client: Add support to configure cluster validation for ingester clients. Failed cluster validations are tracked by `cortex_client_request_invalid_cluster_validation_labels_total` with label `client=ingester`. #10767
```

Comment:
I think we need to improve either the code or the docs (or both) to clarify that `-server.cluster-validation.label` is configuring the _client_.